docs/flask.asciidoc

```diff
@@ -132,11 +132,14 @@ apm.capture_message('hello, world!')
 
 [float]
 [[flask-logging]]
-==== Logging
+==== Shipping Logs to Elasticsearch
 
-Passing `logging=LEVEL` to the ElasticAPM constructor will make the agent automatically log all log messages from Python's built-in `logging` module,
-with the given level or higher.
-`LEVEL` has to be one of the levels defined in the https://docs.python.org/3/library/logging.html#logging-levels[`logging`] module.
+Passing `logging=LEVEL` to the ElasticAPM constructor will make the agent
+automatically ship all log messages from Python's built-in `logging` module
+to Elasticsearch, with the given level or higher. `LEVEL` has to be one of
+the levels defined in the
+https://docs.python.org/3/library/logging.html#logging-levels[`logging`]
+module.
```

Comment:
I struggled with the initial paragraph. Does this seem cleaner to you? If it doesn't, that's fine. Also, these line breaks are probably not correct.

```suggestion
Enable automatic log shipping by passing `logging=LEVEL` to the ElasticAPM constructor.
The agent will then ship all log messages with the given log-level or higher,
from Python's built-in `logging` module to Elasticsearch.
to Elasticsearch. `LEVEL` must be one of the levels defined in the https://docs.python.org/3/library/logging.html#logging-levels[`logging`] module.
```

Reply:
Yes, I liked your suggestion, though it had a typo so I just applied it manually. Thanks!